src/core/annotation.js

```diff
@@ -55,6 +55,22 @@ var ObjectLoader = coreObj.ObjectLoader;
 var FileSpec = coreObj.FileSpec;
 var OperatorList = coreEvaluator.OperatorList;
 
+function hasFlag(flags, bit) {
```

Comment:
Can't we keep this inside the class, but instead use them via private methods? So, `get viewable()` would use `return this._isViewable(this.flags)` and `_isViewable` is a private method of the `Annotation` class. It seems to make sense to me to keep this inside the `Annotation` class as that is essentially where it operates on. All other classes inherit from `Annotation`, so they will also have these private methods.